vdoctl/cmd/drivers.go

```diff
@@ -73,13 +75,28 @@ var driversCmd = &cobra.Command{
 		// Check the vdoDeployment Namespace and confirm if VDO operator is running in the env
 		getVdoNamespace(ctx)
 
-		err, _ := IsVDODeployed(ctx)
+		var isConfigRequired bool
+		configKey := types.NamespacedName{
+			Namespace: VdoCurrentNamespace,
+			Name:      CompatMatrixConfigMap,
+		}
+		cm := v1.ConfigMap{}
+
+		_ = K8sClient.Get(ctx, configKey, &cm)
+		if len(cm.Data) == 0 {
+			isConfigRequired = true
+			color.Yellow("Since you have not configured the Compatibility Matrix so the default values of the matrix will be taken, If you want to configure the matrix manually then you can skip this step and run 'vdoctl configure compatibility matrix")
```

Comment:
lets word this as
VDO will use the default compatibility matrix to determine the versions. If you wish to configure compatibility matrix please run `vdoctl configure compatibility matrix`